CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changes
 4.10 (unreleased)
 -----------------
 
-- Nothing changed yet.
+- Improve test coverage show deprecation warnings when importing modules which
```

Comment:
"Improve test coverage show deprecation warnings ..." isn't quite grammatical English, and I'm not sure exactly what it's meant to say.

Reply:
I split it into two bullet points, which make it hopefully clear what I am trying to say.